src/app/zap-templates/zcl/data-model/chip/ecosystem-information-cluster.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="General"/>
+  <struct name="EcosystemDeviceStruct" isFabricScoped="true" apiMaturity="provisional">
+    <cluster code="0x0750"/>
+    <item fieldId="0" name="DeviceName" type="char_string" optional="true" isFabricSensitive="true" length="64"/>
+    <item fieldId="1" name="DeviceNameLastEdit" type="epoch_us" optional="true" isFabricSensitive="true" default="0"/>
+    <item fieldId="2" name="BridgedEndpoint" type="endpoint_no" optional="true" isFabricSensitive="true"/>
+    <item fieldId="3" name="OriginalEndpoint" type="endpoint_no" optional="true" isFabricSensitive="true"/>
+    <item fieldId="4" name="DeviceTypes" array="true" type="DeviceTypeStruct" isFabricSensitive="true" minLength="1"/>
+    <item fieldId="5" name="UniqueLocationIDs" array="true" type="char_string" isFabricSensitive="true"/>
+    <item fieldId="6" name="UniqueLocationIDsLastEdit" type="epoch_us" isFabricSensitive="true" default="0"/>
+  </struct>
+
+  <struct name="EcosystemLocationStruct" isFabricScoped="true" apiMaturity="provisional">
+    <cluster code="0x0750"/>
+    <item fieldId="0" name="UniqueLocationID" type="char_string" isFabricSensitive="true" length="64"/>
+    <item fieldId="1" name="HomeLocation" type="HomeLocationStruct" optional="true" isFabricSensitive="true"/>
+    <item fieldId="2" name="HomeLocationLastEdit" type="epoch_us" optional="true" isFabricSensitive="true" default="0"/>
+  </struct>
```

Comment:
why it is isFabricScoped?

Reply:
See https://github.com/project-chip/connectedhomeip/pull/34291/files#r1674036642 on why it is Fabric scoped